### PR TITLE
SG -> Spectral

### PR DESCRIPTION
**Pull request by HoeenCoder**
Since this is a HUGE change, I want this checked before its commited.

### CODE REVIEW

File: chat-plugins/uno.js [hunk 1]
```diff
@@ -258,31 +258,31 @@ class UNOgame extends Rooms.RoomGame {
 		return name;
 	}
 
-	sendToRoom(msg, overrideSuppress) {
+	sendToRoom(mSpectral, overrideSuppress) {
```
Comment (HoeenCoder) [inline chat-plugins/uno.js]: Find and replace all can cause this, this is why I PRed first.

File: spectral-plugins/misc.js [hunk 2]
```diff
@@ -188,7 +188,7 @@ exports.commands = {
 		if (!this.can('pmall')) return false;
 		if (!target) return this.parse('/help pmall');
 
-		let pmName = ' SG Server';
+		let pmName = ' Spectral Server';
```
Comment (HoeenCoder) [inline spectral-plugins/misc.js]: We cant use this name unless we register it first.

File: spectral-plugins/misc.js [hunk 3]
```diff
@@ -272,8 +272,8 @@ exports.commands = {
 		return this.sendReplyBox('There ' + (names.length === 1 ? 'is' : 'are') + ' <font color="#24678d"><b>' + names.length + '</b></font> ' + (names.length === 1 ? 'user' : 'users') + ' with the rank <font color="#24678d"><b>' + Config.groups[target].name + '</b></font> currently online.<br />' + names.join(', '));
 	},
 
-	sg: 'spacialgazerepo',
-	sgr: 'spacialgazerepo',
+	Spectral: 'spacialgazerepo',
+	Spectralr: 'spacialgazerepo',
```
Comment (HoeenCoder) [inline spectral-plugins/misc.js]: Not the right way to re-write this

File: spectral-plugins/misc.js [hunk 4]
```diff
@@ -339,54 +339,54 @@ exports.commands = {
 		if (target.length < 2) return this.parse('/help usetoken');
 		target[0] = toId(target[0]);
 		if (target[0] === 'intro') target[0] = 'disableintroscroll';
-		let msg = '';
+		let mSpectral = '';
```
Comment (HoeenCoder) [inline spectral-plugins/misc.js]: more `msg` splitting

File: tells.js [hunk 5]
```diff
@@ -109,12 +109,12 @@ exports.sendTell = function (userid, user) {
  * Store a tell to be received later
  * @param sender	The User object of the sender
  * @param receiver	The target userid
- * @param msg		The message to be send
+ * @param mSpectral		The message to be send
```
Comment (HoeenCoder) [inline tells.js]: `msg` again

File: spectral-plugins/misc.js (hunk 2 above)
Comment (SarcasticResponse) [inline spectral-plugins/misc.js]: I regged this @HoeenCoder

**Pull request merged**